packages/table/src/table.tsx

```diff
@@ -841,8 +855,22 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     private handleRowHeightChanged = (rowIndex: number, height: number) => {
+        const selectedRegions = this.state.selectedRegions;
         const rowHeights = this.state.rowHeights.slice();
-        rowHeights[rowIndex] = height;
+
+        if (Regions.hasFullTable(selectedRegions)) {
+            for (let row = 0; row < rowHeights.length; row++) {
+                rowHeights[row] = height;
```

Comment:
I thought about it, it just seemed like such an abuse of map's actual intention. I can do that if you think it's better, though.